docs/user/build-customization.rst

```diff
@@ -460,23 +460,22 @@ Take a look at the following example:
 
    version: 2
 
+   sphinx:
+      configuration: docs/conf.py
+
    build:
       os: ubuntu-24.04
       tools:
          python: "3.13"
       jobs:
-         create_environment:
+         post_system_dependencies:
             - asdf plugin add uv
             - asdf install uv latest
             - asdf global uv latest
-            - uv venv
+         create_environment:
+            - uv venv "${READTHEDOCS_VIRTUALENV_PATH}"
          install:
-            - uv pip install -r requirements.txt
-         build:
-            html:
-               - uv run sphinx-build -T -b html docs $READTHEDOCS_OUTPUT/html
-
-MkDocs projects could use ``NO_COLOR=1 uv run mkdocs build --strict --site-dir $READTHEDOCS_OUTPUT/html`` instead.
```

Comment:
Seems this is removing the build step completely?

Reply:
It's no longer overriding the build step because it doesn't need to, since `uv` is installing things in the regular RTD virtualenv. Here's a screenshot of a real run of this (from https://github.com/canonical/starbase/pull/330):

![image](https://github.com/user-attachments/assets/21779c6b-469a-4a01-871b-074078748bbe)

Reply:
And the full build log: [starbase-build-log.txt](https://github.com/user-attachments/files/18548241/starbase-build-log.txt)

Reply:
And another example: https://github.com/canonical/craft-application/blob/main/.readthedocs.yaml

[Build log](https://github.com/user-attachments/files/18713231/2791667.txt)

[Output](https://canonical-craft-application.readthedocs-hosted.com/en/latest/)

Reply:
This example still isn't really a common uv use case though? I think we want to show people how to use `uv` in a way that makes it really clear what the steps are?

Reply:
I'm not sure why this wouldn't be a common uv case? This is using uv's [project management](https://docs.astral.sh/uv/#projects) and lock file, where the previous example uses uv as a drop-in replacement for pip but doesn't use [`uv.lock`](https://docs.astral.sh/uv/concepts/projects/layout/#the-lockfile) This also matches quite well to the poetry steps above the example.